integration-test.py

```diff
@@ -159,21 +159,57 @@ def match_distance(actual, expected):
     return (distance, misses)
 
 
-@contextmanager
-def features_in_tile_layer(z, x, y, layer):
+class GatewayTimeout(Exception):
+    def __init__(self, message):
+        self.message = message
```

Comment:
The constructor should be provided for you, and should work this way. In other words, should be similar to:

```python
class GatewayTimeout(Exception):
    pass
```

But if you want to be more explicit it can be left in. In that case, it might not matter but feels safer to add in a call to super:
```python
    super(GatewayTimeout, self).__init__(message)
```

This is test code anyway, so perfectly fine to just leave as is.